src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/product.html.twig

```diff
@@ -160,7 +162,7 @@
         </div>
       {% endif %}
       {% for customization in product.customizations.textCustomizations %}
-        <p><strong>{{ customization.name }}</strong> {{ customization.value }}</p>
+      <p><strong>{{ '%name%:'|trans({'%name%': customization.name}, 'Admin.Ordercustomers.Feature') }} </strong> {{ customization.value }}</p>
```

Comment:
@LouiseBonnard Need wording there, not sure of the domain

Reply:
@NeOMakinG, this one is fine. :-)